### PR TITLE
Visitor can view all listed articles

### PR DESCRIPTION
**Pull request by robin-lillqvist**
# [PT Story](https://www.pivotaltracker.com/story/show/172047601)
## User Story
```
As a Visitor 
In order to choose an article to read 
I would like to be able to see the articles listed on a page
```
## Changes proposed in this pull request:
* Use Redux for state management
* Fetch articles from API
* Add logo to mobile app
## What I have learned working on this feature:
* Ionic setup and using Ionic components
* Refreshed knowledge React and Redux
## Packages/Gems added
- Redux
- React-Redux
- Redux-Thunk
- Axios
## Screenshots (Only if client side story)
Screenshot 2020-03-31 at 14 59 12

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,38 +1,54 @@
...
-import { IonReactRouter } from '@ionic/react-router';
-import { IonApp, IonHeader } from '@ionic/react';
-import Menu from './components/Menu';
-import Footer from './components/Footer';
-import ArticleList from './components/ArticleList';
+import React from 'react'
+import { connect } from 'react-redux'
+// import { Redirect, Route } from 'react-router-dom';
+// import { IonReactRouter } from '@ionic/react-router';
+import { IonApp, IonHeader } from '@ionic/react'
```
Comment (kaylawoodbury) [inline src/App.jsx:3]: I would remove these if they are not currently being used

File: src/index.jsx [hunk 2]
```diff
@@ -1,7 +1,24 @@
...
+import App from './App'
+import { Provider } from 'react-redux'
+import configureStore from './state/store/configureStore'
+import * as serviceWorker from './serviceWorker'
+import axios from "axios";
 
-ReactDOM.render(<App />, document.getElementById('root'));
-serviceWorker.unregister();
+axios.defaults.baseURL = "http://localhost:3000/api";
+
```
Comment (kaylawoodbury) [inline src/index.jsx:11]: be sure to update this to heroku before we do the final merge

**Review by pierre-1 — COMMENTED** [review]
LGTM. Great job guys!

**Review by kaylawoodbury — COMMENTED** [review]
Overall looks good! Left two comments

File: src/index.jsx (hunk 2 above)
Comment (robin-lillqvist) [reply src/index.jsx:11]: done

**Comment by ghost** [comment]
Looks good

File: src/App.jsx (hunk 1 above)
Comment (robin-lillqvist) [reply src/App.jsx:3]: Done!

**Pull request merged**